mods/ts/rules/defaults.yaml

```diff
@@ -1160,30 +1156,37 @@
 			VerticalTopOffset: 768
 	EditorTilesetFilter:
 		Categories: Wall
+	Gate:
+		OpeningSound: gateup1.aud
+		ClosingSound: gatedwn1.aud
+		BlocksProjectilesHeight: 768
+		PauseOnCondition: empdisable
 
 ^Gate_A:
 	Inherits: ^Gate
 	Inherits@shape: ^3x1Shape
 	Health:
-	Gate:
+	Building:
 		Dimensions: 3,1
 		Footprint: xxx
 	WithGateSpriteBody:
 		WallConnections: -1,0, 3,0
 	LineBuildNode:
 		Connections: -1,0, 1,0
+	Gate:
 
 ^Gate_B:
 	Inherits: ^Gate
 	Inherits@shape: ^1x3Shape
 	Health:
-	Gate:
+	Building:
 		Dimensions: 1,3
 		Footprint: x x x
 	WithGateSpriteBody:
 		WallConnections: 0,-1, 0,3
 	LineBuildNode:
 		Connections: 0,-1, 0,1
+	Gate:
```

Comment:
`^Gate_B` already inherits `^Gate`, which has the `Gate` trait, so this is redundant.

Reply:
For the record: those redundancies came from the upgrade rule, I just didn't check for redundancies (due to lazyness, mostly).